ds_judgements_public_ui/sass/includes/_footer.scss

```diff
@@ -52,10 +52,10 @@
 
   li {
     list-style: none;
-    margin-bottom: calc($spacer__unit / 4);
+    margin-bottom: calc($spacer__unit * 1.6);
```

Comment:
More for my understanding in how these decisions get made: 

are the scaling values of 1.6 and 1.4 specific values from some guidance, or it's just what you settled on after tweaking and seeing what looks good?

Reply:
I've looked into WCAG but there is no clear definition or formula to follow.
So used designer's eye to judge it really @anthonyhashemi . I will be ask DAC who will run an external accessibility report, s will come back to this. Also WCAG2.2 has some new rules coming down the track for target areas. so sems prudent to improve this as a tool I used raised it as a fail.